pkg/kubelet/types/pod_update.go

```diff
@@ -92,9 +92,8 @@ func GetValidatedSources(sources []string) ([]string, error) {
 			return []string{FileSource, HTTPSource, ApiserverSource}, nil
 		case FileSource, HTTPSource, ApiserverSource:
 			validated = append(validated, source)
-			break
 		case "":
-			break
```

Comment:
`break` exist the loop fast instead of continuing.
what is the error here?

Reply:
`break` exits the switch statement here and not the for loop, in this case `break` is redundant

see:

https://play.golang.org/p/KcdweIOKTYK

and https://golang.org/ref/spec#Break_statements

It's unclear what the intention of the code, ideally a label would be used if the intention was to break out of the for loop

Reply:
my mistake. this makes them redundant indeed.